### ISSUE TITLE
`payable` keyword not working, getting error "Syntax error: unexpected token p"

### ISSUE THREAD

**Issue by miguelmota** [duaraghav8/Ethlint#244]
```solidity
pragma solidity >0.4.99 <0.6.0;

contract Example {
    address payable public alice;

    constructor(address payable _alice) public {
      alice = _alice;
    }
}
```

```bash
solium -f contracts/Example.sol

packages/channel-contracts/contracts/Example.sol
  4:21    error    Syntax error: unexpected token p

✖ 1 error found.
```

**Comment by duaraghav8** [comment]
@miguelmota Thanks for reporting this. I'm not sure whether `payable` can be used with `address` (or maybe my solidity knowledge is outdated), could you please point me to any official examples or doc on this?

**Comment by miguelmota** [comment]
@duaraghav8 it's part of 0.5 
https://solidity.readthedocs.io/en/v0.5.0/050-breaking-changes.html

**Comment by duaraghav8** [comment]
solved in `v1.2.0`

This linter is now being published simultaneously to npm packages `solium` and `ethlint`. Although not mandatory, I highly recommend that you move to `ethlint`.

See [changelog](https://github.com/duaraghav8/Ethlint/blob/master/CHANGELOG.md#120-2018-12-25-santa)
Please let me know in case any of the 0.5 features are still causing trouble.
